src/infi/systray/traybar.py

```diff
@@ -122,15 +122,25 @@ def shutdown(self):
         PostMessage(self._hwnd, WM_CLOSE, 0, 0)
         self._message_loop_thread.join()
 
-    def update(self, icon=None, hover_text=None):
-        """ update icon image and/or hover text """
+    def update(self, icon=None, hover_text=None, menu_options=None):  #  "menu_options=None" added to be allow the update of the menu options

```

Comment:
Please remove the comment. It's making the line too long and it isn't necessary.

Reply:
Done!